src/main/java/service/impl/ReportCreatorImpl.java

```diff
@@ -0,0 +1,35 @@
+package service.impl;
+
+import static db.Storage.fruitStorage;
```

Comment:
You should create dao interface and its implementation to connect your storage. It's bad practice to access storage directly